modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/UserRow.es.js

```diff
@@ -23,7 +23,7 @@ export default withRouter(
 	({creator = {}, statistics, hasCompanyMx, companyName}) => {
 		return (
 			<Link
-				className="align-items-center border-0 btn btn-block btn-secondary d-flex position-relative questions-user text-left text-md-right"
+				className="align-items-center border-0 btn btn-block btn-secondary d-flex position-relative questions-creator-success questions-user text-left text-md-right"
```

Comment:
No need to add this class

```suggestion
				className="align-items-center border-0 btn btn-block btn-secondary d-flex position-relative questions-user text-left text-md-right"
```

Reply:
Basic do not touch this classNames because we can use questions-user instead